### ISSUE TITLE
Uncaught TypeError: target.cloneNode is not a function

### ISSUE THREAD

**Issue by EnesSefaK** [facebook/nuclide#1116]
[Enter steps to reproduce:]

1. ...
2. ...

**Atom**: 1.16.0 x64
**Electron**: 1.3.13
**OS**: Ubuntu 17.04
**Thrown From**: [nuclide](https://github.com/facebook/nuclide) package 0.219.0

### Stack Trace

Uncaught TypeError: target.cloneNode is not a function

```
At /home/enes/.atom/packages/nuclide/node_modules/rxjs/bundles/Rx.min.js:33

TypeError: target.cloneNode is not a function
  at FileTreeEntryComponent._onDragStart (/packages/nuclide/pkg/nuclide-file-tree/components/FileTreeEntryComponent.js:415:29)
  at a.__tryOrUnsub (/packages/nuclide/node_modules/rxjs/bundles/Rx.min.js:33:10)
  at a.next (/packages/nuclide/node_modules/rxjs/bundles/Rx.min.js:31:454)
  at a._next (/packages/nuclide/node_modules/rxjs/bundles/Rx.min.js:30:320)
  at a.next (/packages/nuclide/node_modules/rxjs/bundles/Rx.min.js:29:506)
  at /packages/nuclide/node_modules/rxjs/bundles/Rx.min.js:70:230)

```

### Commands

```
```

### Non-Core Packages

```
atom-beautify 0.29.23 
auto-indent 0.5.0 
autocompiler-pug-jade 2.0.1 
busy-signal 1.4.1 
emmet 2.4.3 
file-icons 2.1.3 
file-type-icons 1.3.3 
intentions 1.1.2 
language-pug 0.0.21 
linter 2.1.4 
linter-sass-lint 1.7.4 
linter-ui-default 1.2.3 
npm-install 4.0.4 
nuclide 0.219.0 
```

**Comment by shushz** [comment]
Thanks for the report - we'll hotfix this.

**Comment by shushz** [comment]
This is fixed in v0.220.0
